CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# CHANGELOG
+Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+### Added
+- Github workflow for changelog verification ([#4085](https://github.com/opensearch-project/OpenSearch/pull/4085))
```

Comment:
@dblock This was the issue that I was pointing to with backported PRs. We will have to manually intervene and update the previous version as well. 

Or know before hand which versions we will backport to as a contributor and add that to all the listed versions.

Reply:
Created a PR to track 2.x changes here: https://github.com/opensearch-project/OpenSearch/pull/4297
Will merge in sequence.